DispatchGroup/ViewController.swift

```diff
@@ -59,7 +59,7 @@ class ViewController: UIViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
-//        dispatchGroupCall()
+        //        dispatchGroupCall()
         operationQueue()
     }
     
```

Comment:
imageView1.sd_setImage(with: URL(string: (self.result1?.imageResults[0].image.src)!))
                imageView2.sd_setImage(with: URL(string: (self.result2?.imageResults[0].image.src)!))
                imageView3.sd_setImage(with: URL(string: (self.result3?.imageResults[0].image.src)!))

This alread downloads and sets the image to imageview automatically in background